native/src/snappyclient/cpp/impl/ClientService.cpp

```diff
@@ -1885,3 +1891,9 @@ bool ClientService::handleException(const char* op, bool tryFailover,
   updateFailedServersForCurrent(failedServers, true, te);
   return true;
 }
+void ClientService::getSSLPropertyValue(std::string& propertyName, std::string& value){
+  sslParams.getSSLPropertyValue(propertyName, value);
+}
```

Comment:
Change to return by value instead of by reference.

Reply:
Resolved